politeiawww/user/cockroachdb/cockroachdb_test.go

```diff
@@ -0,0 +1,362 @@
+	}
+	// try updating the same session, this should not return an error but
+	// update the record.
+	expected.Values += " / update"
+	expected.UserID = uuid.New()
+	err = db.SessionSave(expected)
+	if err != nil {
+		t.Errorf("SessionSave() #2 returned an error: %v", err)
+	}
+
+	us2, err := db.SessionGetById(expected.ID)
+	if err != nil {
+		t.Errorf("SessionGetById() returned an error: %v", err)
+	}
+
+	if us2.Values != expected.Values {
+		t.Errorf("got Values: %v, want: %v", us2.Values, expected.Values)
+	}
+	if us2.UserID != expected.UserID {
+		t.Errorf("got UserID: %v, want: %v", us2.UserID, expected.UserID)
+	}
+}
+
+func TestSessionGetById(t *testing.T) {
+	db := connectToTestDB(t)
+	expected := user.Session{
+		ID:     uuid.New().String(),
+		UserID: uuid.New(),
+		Values: "TestSessionDeleteById()",
+	}
+
+	// insert a session
+	err := db.SessionSave(expected)
+	if err != nil {
+		t.Errorf("SessionSave() returned an error: %v", err)
+	}
+
+	// get the Session we just inserted
+	us, err := db.SessionGetById(expected.ID)
+	if err != nil {
+		t.Errorf("SessionGetById() returned an error: %v", err)
+	}
+
+	var model Session
+	err = db.userDB.Where("id = ?", expected.ID).First(&model).Error
+	if err != nil {
+		t.Errorf("First() returned an error: %v", err)
+	}
+
+	if expected != *us {
+		t.Errorf("got session: %v, want: %v", us, expected)
+	}
+}
+
+func TestSessionGetByIdWithNoRecord(t *testing.T) {
+	db := connectToTestDB(t)
+	expected := user.Session{
+		ID:     uuid.New().String(),
+		UserID: uuid.New(),
+		Values: "TestSessionGetByIdWithNoRecord()",
+	}
+
+	// try to get a session that does not exist
+	_, err := db.SessionGetById(expected.ID)
+	if err == nil {
+		t.Error("SessionGetById() did not return an error")
+	}
+	if err != user.ErrSessionDoesNotExist {
+		t.Errorf("got error: %v, want: %v", err, user.ErrSessionDoesNotExist)
+	}
+}
+
+func TestSessionDeleteById(t *testing.T) {
+	db := connectToTestDB(t)
+	var err error
+	sa := []user.Session{
+		{
+			ID:     uuid.New().String(),
+			UserID: uuid.New(),
+			Values: "TestSessionDeleteById() / 1",
+		},
+		{
+			ID:     uuid.New().String(),
+			UserID: uuid.New(),
+			Values: "TestSessionDeleteById() / 2",
+		},
+		{
+			ID:     uuid.New().String(),
+			UserID: uuid.New(),
+			Values: "TestSessionDeleteById() / 3",
+		},
+	}
+
+	for idx, s := range sa {
+		// insert a session
+		err = db.SessionSave(s)
+		if err != nil {
+			t.Errorf("idx: %v, SessionSave() returned an error: %v", idx, err)
+		}
+	}
+
+	removed := []int{1}
+	for _, idx := range removed {
+		// delete the session
+		err = db.SessionDeleteById(sa[idx].ID)
+		if err != nil {
+			t.Errorf("idx: %v, SessionDeleteById() returned an error: %v", idx, err)
+		}
+		// make sure the deleted session is gone
+		var model Session
+		err = db.userDB.Where("id = ?", sa[idx].ID).First(&model).Error
+		if err != gorm.ErrRecordNotFound {
+			t.Errorf("idx: %v, got error: %v, want: %v", idx, err, gorm.ErrRecordNotFound)
```

Comment:
80 column limit.  There are a couple instances of this in the test file.  Please go through and fix them.

```suggestion
			t.Errorf("idx: %v, got error: %v, want: %v",
				idx, err, gorm.ErrRecordNotFound)
```

Reply:
Done.